<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IhmAgents</class>
 <widget class="QWidget" name="IhmAgents">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidgetAgents">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>891</width>
     <height>301</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Prénom</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nom</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>N° de badge</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Statut</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Modification</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>20</y>
     <width>591</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Filtres : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>nom :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxNom">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>statut :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxStatut">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>badge :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxBadge">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAppliquer">
        <property name="text">
         <string>Appliquer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>400</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="pushButtonActualiser">
      <property name="text">
       <string>Actualiser</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonNouveau">
      <property name="text">
       <string>Nouveau</string>
      </property>
      <property name="icon">
       <iconset resource="Images.qrc">
        <normaloff>:/img/images/ajouterIcon.png</normaloff>:/img/images/ajouterIcon.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
